docs/creating.rst

```diff
@@ -59,6 +65,10 @@ Creating or Extending Validator Classes
 
     :argument str version: a version for the new validator class
 
+    :argument jsonschema.TypeChecker type_checker: an instance
+        of :class:`TypeChecker`. If unprovided, the existing
```

Comment:
I try not to repeat instance-of-class kind of info in the actual prose (because it's already in the `:argument:` role). It's easy to get around with just "a type checker.", even though the class name is pretty similar in this caes.